textattack/trainer.py

```diff
@@ -0,0 +1,884 @@
+import collections
```

Comment:
Is there a test for this file (adv training in general) during builds?

Reply:
We do have one command line test. But we should add API version next.